controller/src/main/java/io/pravega/controller/server/ControllerServiceMain.java

```diff
@@ -139,23 +137,24 @@ protected void run() throws Exception {
                     // stop ControllerServiceStarter.
                     if (sessionExpiryFuture.isDone()) {
                         zookeeperMetrics.reportZKSessionExpiration();
-                        log.info("ZK session expired");
+                        log.debug("ZK session expired");
```

Comment:
we should log this at INFO level, right? Is there a different log which indicates a session expiry?

Reply:
It would be good to have a single log that explains that the Controller Service is stopping and why... changed this so that we log accordingly...